mm2src/coins/my_tx_history_v2.rs

```diff
@@ -134,10 +133,6 @@ pub trait TxHistoryStorage: Send + Sync + 'static {
     ) -> Result<GetHistoryResult, MmError<Self::Error>>;
 }
 
-pub trait DisplayAddress {
-    fn display_address(&self) -> String;
-}
-
 impl DisplayAddress for Address {
```

Comment:
Could we move DisplayAddress impls from this source (to utxo_hd_wallet.rs maybe?)

Reply:
Moved utxo addresses to utxo hd wallet https://github.com/KomodoPlatform/komodo-defi-framework/pull/2348/commits/e7015abfda5f37ebbd6654e787517ed5163fe8d1